src/V3Waiver.cpp

```diff
@@ -31,6 +31,7 @@ void V3Waiver::addEntry(V3ErrorCode errorCode, const std::string& filename,
           << str.substr(0, pos);
     if (pos != std::string::npos) entry << "*";
     entry << "\"";
+    const VerilatedLockGuard lg{s_mutex};
```

Comment:
Put the lock changes into your other patch, keeping this one just benign comments?

Anyhow without a VL_MT comment on the function being locked there's no reason to add this yet.